service/reportsStorage/handlers.go

```diff
@@ -0,0 +1,188 @@
+package reportsStorage
+
+import (
+	"github.com/go-openapi/runtime/middleware"
+	"github.com/rs/zerolog"
+
+	"github.com/iryonetwork/wwm/gen/reports/models"
+	"github.com/iryonetwork/wwm/gen/reports/restapi/operations"
```

Comment:
could not import github.com/iryonetwork/wwm/gen/reports/restapi/operations (cannot find package "github.com/iryonetwork/wwm/gen/reports/restapi/operations" in any of:
	/tmp/tmp.qkYKz9nbyP/src/github.com/iryonetwork/wwm/vendor/github.com/iryonetwork/wwm/gen/reports/restapi/operations (vendor tree)
	/usr/local/go/src/github.com/iryonetwork/wwm/gen/reports/restapi/operations (from $GOROOT)
	/tmp/tmp.qkYKz9nbyP/src/github.com/iryonetwork/wwm/gen/reports/restapi/operations (from $GOPATH))